example/PennFudanPed/Makefile

```diff
@@ -10,3 +10,12 @@ download-data:
 	curl -o data/pfp.zip https://www.cis.upenn.edu/~jshi/ped_html/PennFudanPed.zip
 	unzip data/pfp.zip -d data
 	rm -rf data/pfp.zip
+
+.POHNY: prepare-e2e-data
```

Comment:
the example code is for the common users, should we move the prepare-e2-data section into the e2e folders?  The users feel confused with `10.131.0.1`?